docs/manual/working/scalaGuide/main/json/code/ScalaJsonCombinatorsSpec.scala

```diff
@@ -197,8 +199,8 @@ class ScalaJsonCombinatorsSpec extends Specification {
       val json: JsValue = sampleJson
 
       json.validate[Place] match {
-        case s: JsSuccess[Place] => {
-          val place: Place = s.get
+        case JsSuccess(place, _) => {
+          val _: Place = place
```

Comment:
because `place` is already being extracted, we can remove this line.

Reply:
Removing would raise unused warning, whereas it's interesting to have such test as example how to pattern-match on `JsResult`

Reply:
yes, correct. Thanks for the explanation.